recipes/rospkg/recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rospkg" %}
+{% set version = "1.1.9" %}
+{% set sha256 = "26a06e5ee07f305bc5e5ac2855fb779fc1207797bc4f9ce303f6d88e4c568fcc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ros-infrastructure/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  entry_points:
+    - rosversion = rospkg.rosversion:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - catkin_pkg
```

Comment:
The following two dependencies don't seem to be required.

Reply:
removed them from host. let's see if it builds fine.

Reply:
It did :+1:

Reply:
`host` is basically what `setup.py` requires. `conda skeleton pypi` sadly cannot infer this.